core/core-documentation.el

```diff
@@ -85,19 +88,99 @@
+                                    (substring (toc-org-hrefify-gh
+                                                (replace-regexp-in-string
+                                                 toc-org-tags-regexp
+                                                 ""
+                                                 heading))
+                                               ;; Remove # prefix added by `toc-org-hrefify-gh'.
+                                               1))))))))
+
+(defun spacemacs//reroot-links ()
+  "Find the links that start with https://github.com/syl20bnr/spacemacs/blob/
+and end with .org{#an-optional-heading-link} (i.e the links between the local org files).
+Change their root to http://spacemacs.org/ so the links will point at files located on the site.
+For the file to file links to work properly the exported org files should be processed with
+the `spacemacs//org-heading-annotate-custom-id' function."
+  (let ((git-url-root-regexp
+         (concat "\\[\\[[\\s]*\\(https\\:\\/\\/github\\.com\\/syl20bnr"
+                 "\\/spacemacs\\/blob\\/[^/]+\\/\\)[^]]+\\(\\.org\\).*$"))
+        (site-url "http://spacemacs.org/")
+        (site-doc-postf ".html"))
+    (progn (goto-char (point-min))
+           (while (re-search-forward git-url-root-regexp nil t)
+             (progn (replace-match site-url nil t nil 1)
+                    (replace-match site-doc-postf nil t nil 2))))))
+
+(defun spacemacs//add-org-meta-readtheorg-css (filename)
+  (let* ((head-css-extra-readtheorg-head (concat
+                                          "#+HTML_HEAD_EXTRA:"
+                                          "<link rel=\"stylesheet\" "
+                                          "type=\"text/css\" "
+                                          "href=\""))
+         (head-css-extra-readtheorg-tail "css/readtheorg.css\" />\n"))
+    (progn (goto-char (point-min))
+           (delete-matching-lines "\\+HTML_HEAD_EXTRA\\:.*\\/css\\/readtheorg\\.css")
+           (goto-char (point-min))
+           (if (search-forward "#+TITLE:" nil t nil)
+               (beginning-of-line 2)
+             (error (format "Can't find #+TITLE: in %s"
+                            (buffer-file-name))))
+           (insert (concat head-css-extra-readtheorg-head
+                           (f-relative user-emacs-directory
+                                       (file-name-directory filename))
+                           head-css-extra-readtheorg-tail)))))
+
+(defun spacemacs//pub-doc-html-advice (origfunc &rest args)
+  (save-current-buffer
+    (save-excursion
+      (let* ((filename (car (nthcdr 1 args)))
+             (visitingp (find-buffer-visiting filename)))
+        ;; Temporary "unvisit" the visited org files.
+        (when visitingp (with-current-buffer visitingp (setq buffer-file-name nil)))
+        (with-temp-buffer
+          (save-match-data
+            (insert-file-contents filename t)
+            (spacemacs//add-org-meta-readtheorg-css filename)
```

Comment:
You can add a comment here to specify that any preprocess function should be added here.